operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -60,7 +60,6 @@ def __init__(self, plant, max_power_filter = 0.85, wind_bin_thresh=2):
         
         self._max_power_filter = max_power_filter # Parameter used for bin-based filtering
         self._wind_bin_thresh = wind_bin_thresh
-        self._reanal = ['merra2', 'erai', 'ncep2'] # Reanalysis products to consider
```

Comment:
Sorry, looks like an improvement to turbine gross energy snuck in here. The idea is to not necessarily run all 3 reanalysis products but, similar to the Monte Carlo AEP method, allow the user to specify which reanalysis products to run.